### ISSUE TITLE
Inplace updating a sequence with an anchor unsets the anchor

### ISSUE THREAD

**Issue by ashwin1dd** [mikefarah/yq#1269]
**Describe the bug**
Appending to a sequence inplace unsets its anchor.
It shouldn't affect the anchor at all, and should've only affected the sequence.

Version of yq: 4.25.3
Operating system: mac
Installed via: homebrew

**Input Yaml**
Concise yaml document(s) (as simple as possible to show the bug, please keep it to 10 lines or less)
data1.yml:
```yaml
 a: &hello
   - x
   - y
 b: *hello
```

**Command**
The command you ran:
```
yq -i '.a += ["z"]' data1.yml
```

**Actual behavior**

```yaml
a:
  - x
  - y
  - z
b: *hello
```

**Expected behavior**

```yaml
a: &hello
  - x
  - y
  - z
b: *hello
```

**Additional context**

**Comment by mikefarah** [comment]
Yep that's a bug - will fix in the next release.

As a workaround, you can do the following:

```bash
yq '.a = .a + "z"' file.yaml
```

**Comment by mikefarah** [comment]
Sorry github autoclosed - will close when the fix is actually released..

**Comment by mikefarah** [comment]
Fixed in v4.26.1
